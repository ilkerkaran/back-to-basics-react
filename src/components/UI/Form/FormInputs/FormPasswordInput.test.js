import React from 'react';
import renderer from 'react-test-renderer';
import Password from './FormPasswordInput';


describe('<Password /> snapshot ', () => {
  const inputConfig = {
    inputType: 'password',
    inputName: 'password',
    label: 'Password',
    isRequired: true,
    register: () => {},
    errors: {}
  };


  it('renders correctly', () => {
    const tree = renderer
      .create(<Password inputConfig={inputConfig}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
