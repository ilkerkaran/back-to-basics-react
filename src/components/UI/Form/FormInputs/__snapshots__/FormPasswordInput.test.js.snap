// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Password /> snapshot  renders correctly 1`] = `
<div>
  <input
    className="Input"
    name="password"
    placeholder="Password"
    type="password"
  />
  <div
    style={
      Object {
        "color": "darkred",
      }
    }
  />
</div>
`;
